config/ecs/config.go

```diff
@@ -25,13 +37,13 @@ func init() {
 			"log_configuration[*].s3_bucket_name": config.Reference{
```

Comment:
```suggestion
			"log_configuration.s3_bucket_name": config.Reference{
```

it would be good to search for `[*]` within config dir